src/app/markers.ts

```diff
@@ -68,9 +75,16 @@ export function roundMarkerFromRoundFlower(
     isBlooming: flower.isBlooming,
     isNest: false,
     canVisit,
+    incompatibleFlower,
+    tip: lastVisitedIncompatible ? 'I don\'t like this flower. No Pollen collected' : null
```

Comment:
I'm not sure we need the "No Pollen collected" part since we will be showing a different animation and the pollen count won't increase.

Reply:
Should we keep it until the animation is added and then that part of the tip can be removed ?